src/main/java/homework_4/custom_annotation/db_annotation/Main.java

```diff
@@ -0,0 +1,113 @@
+package homework_4.custom_annotation.db_annotation;
+
+import java.lang.reflect.Field;
+import java.sql.*;
+import java.util.HashMap;
+
+public class Main {
```

Comment:
Opt: Doesn't work on my machine without drivers, but I understand the meaning. Quite cool, but I can't check does it work or not. Provide screenshots if possible

Reply:
![image](https://user-images.githubusercontent.com/60847517/134045639-d2ab7cfe-1326-4c24-9d5f-a5cb77f18940.png)

Reply:
Nice!